src/js/binaryen.js-post.js

```diff
@@ -1563,19 +1563,3 @@ Module['setDebugInfo'] = function(on) {
 Module['setAPITracing'] = function(on) {
   return Module['_BinaryenSetAPITracing'](on);
 };
-
-// Instantiates a new unique instance of the API with its own memory etc.
-Module['instantiate'] = instantiate;
```

Comment:
This functionality is lost because the factory function is named `Binaryen` that also has a local named `Binaryen`. Thus, the function cannot be referenced anymore.

```js
var Binaryen = function(Binaryen) {
  Binaryen = Binaryen || {};
```

That's not a problem, though, and it might be better to incorporate `MyModuleInstance.instantiate` into `MODULARIZE_INSTANCE` in a WASM-compatible way, if at all.